src/firefly/java/edu/caltech/ipac/firefly/server/db/spring/mapper/SqlResultSetUtil.java

```diff
@@ -0,0 +1,91 @@
+package edu.caltech.ipac.firefly.server.db.spring.mapper;
+
+import edu.caltech.ipac.firefly.server.util.Logger;
+import edu.caltech.ipac.table.DataGroup;
+import edu.caltech.ipac.table.DataObject;
+import edu.caltech.ipac.table.DataType;
+import edu.caltech.ipac.util.CollectionUtil;
+import javax.sql.DataSource;
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+
+
+
+public class SqlResultSetUtil {
```

Comment:
Common code that's already in EmbeddedDbUtil.  Either use that, or move that logic to a file that more generic.. like DBUtil.java